src/api.c

```diff
@@ -536,10 +536,16 @@ STATIC int cgroup_parse_rules_options(char *options, struct cgroup_rule * const
 		cmp_len = min(strlen(stok_buff), strlen(CGRULE_OPTION_IGNORE));
 		if (strlen(stok_buff) == strlen(CGRULE_OPTION_IGNORE) &&
 		    strncmp(stok_buff, CGRULE_OPTION_IGNORE, cmp_len) == 0) {
-			rule->is_ignore = true;
+			rule->is_ignore = 1;
 			continue;
 		}
 
+		cmp_len = min(strlen(stok_buff), strlen(CGRULE_OPTION_IGNORE_RT));
+		if (strlen(stok_buff) == strlen(CGRULE_OPTION_IGNORE_RT) &&
+		    strncmp(stok_buff, CGRULE_OPTION_IGNORE_RT, cmp_len) == 0) {
+			rule->is_ignore = 2;
+			continue;
+		}
 		/*
 		 * "ignore" is the only currently supported option.
```

Comment:
This comment is now out of date.  We can probably just delete it since the `cgroup_err()` below clearly states why there was a problem.

Reply:
Sure, will remove the comments.